app/views/mods_panel.py

```diff
@@ -1917,9 +1943,10 @@ def recreate_mod_list_and_sort(
             None
         """
         # TODO: Fix this or just get rid of it
-        sorted_uuids = uuids
+        # sorted_uuids = uuids
         # if key != ModsPanelSortKey.NOKEY:
         #    sorted_uuids = sorted(uuids, key=key)
```

Comment:
Should be cleaned up and removed.

Reply:
Thanks for looking into this! I'm working on addressing these issues.